src/Rules/Properties/PropertiesInInterfaceRule.php

```diff
@@ -25,6 +28,10 @@ public function processNode(Node $node, Scope $scope): array
 			return [];
 		}
 
+		if (PHP_VERSION_ID >= self::PHP_8_4_VERSION_ID) {
```

Comment:
This logic needs to use PhpVersion - you inject that into the constructor, and you ask a new method `supportsPropertyHooksInInterfaces`.